### PR TITLE
Added a Julia set benchmark

### PR DESCRIPTION
**Pull request by xvallspl**

### CODE REVIEW

**Comment by amadio** [comment]
Thanks! I will test and merge.

File: bench/julia.cc [hunk 1]
```diff
@@ -0,0 +1,155 @@
...
+
+int main(int argc, char *argv[])
+{
+    double xmin = -2, xmax = 2;
+    double ymin = -2, ymax = 2;
+
+    size_t nx = 2400, ny = 2025, max_iter = 200;
+    double cr = 0.285, ci = 0.01;
+    std::cout<<cr<<std::endl;    
+    std::cout<<ci<<std::endl;
```
Comment (amadio) [inline bench/julia.cc]: You have trailing whitespace here. Please remove unnecessary print outs.

File: bench/julia.cc [hunk 2]
```diff
@@ -0,0 +1,155 @@
...
+
+   if(argc>1) {
+      if(argc!=3) {
+        std::cerr<<"Error. Incorrect number of parameters for the c parameter in the quadratic polinomial z\x00B2+c. Usage: julia [<realComponent> <imaginary component>] "<<std::endl;
+        exit(-1);
+      } else {
+        cr = atof(argv[1]);
+        ci = atof(argv[2]);
+      }
+   }
```
Comment (amadio) [inline bench/julia.cc]: Please format like the rest of the code (4 spaces rather than 3, and spaces after `if`, etc).

File: bench/julia.cc [hunk 3]
```diff
@@ -0,0 +1,155 @@
...
+
+    size_t nx = 2400, ny = 2025, max_iter = 200;
+    double cr = 0.285, ci = 0.01;
+    std::cout<<cr<<std::endl;    
+    std::cout<<ci<<std::endl;    
+    unsigned char *image = new unsigned char[nx*ny];
+
+   if(argc>1) {
+      if(argc!=3) {
+        std::cerr<<"Error. Incorrect number of parameters for the c parameter in the quadratic polinomial z\x00B2+c. Usage: julia [<realComponent> <imaginary component>] "<<std::endl;
```
Comment (amadio) [inline bench/julia.cc]: This line is too long.

File: bench/julia.cc [hunk 4]
```diff
@@ -0,0 +1,155 @@
...
+        cr = atof(argv[1]);
+        ci = atof(argv[2]);
+      }
+   }
+      
+
+    /* single precision */
+
+    bench_julia<float>(xmin, xmax, nx, ymin, ymax, ny,
+                            max_iter, image, "float", cr, ci);
```
Comment (amadio) [inline bench/julia.cc]: Misaligned parameters.

**Comment by xvallspl** [comment]
Sorry I missed the comments. Let's see if now.

**Comment by amadio** [comment]
Thanks Xavi.

**Pull request merged**